charts/nextcloud/files/app-management.sh.gotmpl

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+{{- range $app,$appConfig := .Values.nextcloud.appManagement.apps }}
+echo "== manage app: {{ $app }} ==";
+
+{{- if $appConfig.enabled }}
+echo "enable:"
+/var/www/html/occ app:install {{ $app }}
+/var/www/html/occ app:enable {{ $app }}
+
+# config of app
+echo "config:"
+{{- range $key, $value := $appConfig.config }}
+/var/www/html/occ config:app:set -n {{ $app }} {{ $key }} --value="{{ $value }}"
```

Comment:
In some cases you want to be able to specify the value type, otherwise `"true"` will be set as a string and not as a boolean.